### PR TITLE
Add isTimeout to static API

### PR DESCRIPTION
**Pull request by EyalPerry**
## Related Issue
Issue: https://github.com/axios/axios/issues/2494

## PR Description
Added the isTimeout function to the static API

### CODE REVIEW

File: test/specs/timeout.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  });
+
+  afterEach(function() {
+    jasmine.Ajax.uninstall();
+  });
+
+    it('resulting error should indicate rejection when the request times out', function (done) {
+      //TODO note this request never rejects, I do not know why
+      axios.get('/foo', {timeout: 100}).catch(function (thrown) {
+        expect(error.axiosTimeout).toBe(true);
```
Comment (yasuf) [inline test/specs/timeout.spec.js]: where is error coming from here?

**Comment by jasonsaayman** [comment]
@EyalPerry please can you check the tests, they are failing.

**Comment by EyalPerry** [comment]
@jasonsaayman Oh god, this is embarassing. how did this slip my mind for 8 months 🤦 
I'd love to. not able to really get down to what's wrong though.
Not sure what to do with the error, doesn't seem directly related to the feature.
Any chance you could look and give me a hint?

File: test/specs/timeout.spec.js (hunk 1 above)
Comment (EyalPerry) [reply test/specs/timeout.spec.js:13]: @yasuf not sure what you mean?

**Comment by jasonsaayman** [comment]
@EyalPerry not a problem I mean pull requests are only now getting done, this is whats in the log:

```
Firefox 63.0.0 (Linux 0.0.0) timeout resulting error should indicate rejection when the request times out FAILED
	Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL. in node_modules/jasmine-core/lib/jasmine-core/jasmine.js (line 4470)
	attempt/timeoutId<@node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4470:23
```

**Comment by jasonsaayman** [comment]
Hi,

Thank you for this pull request, however, it seems that it has gone stale and is either far out of date with the current master head or has unresolved review issues. Should you feel that this pull request is still relevant please submit a new pull request.

Thanks

**Pull request closed without merging**